src/browser/browser.js

```diff
@@ -255,8 +258,22 @@ define((require, exports, module) => {
         onClose: item => webViewersCursor.update(closeTab(item)),
         onOpen: item => webViewersCursor.update(addTab(item))
       })
-    ]);
-  });
+    ]),
+    DOM.div({
```

Comment:
> for the animation.

Can you add comment somewhere then so it's clear. Also in long term we should find a better way to handle that not just have everything in DOM at all times. I imagine react users have figured that out, I'll look into that.